cla_backend/apps/cla_butler/tasks.py

```diff
@@ -78,6 +80,18 @@ def get_eligible_cases(self):
 
         return Case.objects.filter(modified__lte=three_years).exclude(outcome_code__in=self.OUTCOME_CODES)
 
+    def get_cases_without_personal_details(self):
+        """
+        Returns queryset of personal detail's id that are none
+        and do not have notes and provider details
+        """
+
+        return (
+            Case.objects.filter(personal_details_id__exact=None)
```

Comment:
is this definitely catching only those that have empty notes and empty provider notes, ie it is cumulative? Can you add tests that see what happens when you have one or the other and both? I think the tests should be extended.